### PR TITLE
[docs] mention direnv in contributing guideline

### PR DESCRIPTION
**Pull request by jarvisluong**
# Why

When I setup local env to run expo, I noticed that direnv is required but not mentioned in the Contributing guideline

### CODE REVIEW

**Review by cruzach — CHANGES_REQUESTED** [review]
(Editing my prior comment, I was mistaken)

Thanks for adding this!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Manual smoke tests are included in `apps/native-component-list`, this is a good
 
    - Downloads submodules (like `react-native`) with `git submodule update --init`
    - Fetches files with [_git lfs_](https://git-lfs.github.com), which we use for big native libraries like Google Mobile Vision. Note: you must have `git lfs` already installed.
+   - Setup local environment variable and aliases with [_direnv_](https://direnv.net/). Note: you must have `direnv` already installed
```
Comment (cruzach) [inline CONTRIBUTING.md:38]:
```suggestion
   - Ensures [direnv](https://direnv.net/) is installed
```

**Review by cruzach — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (jarvisluong) [reply CONTRIBUTING.md:38]: You're right!

**Comment by cruzach** [comment]
@jarvisluong don't worry about the failing docs CI job, that's on our end

**Pull request merged**